src/utils/IO/system_checks.jl

```diff
@@ -113,7 +109,12 @@ Total sum of system generator ratings.
 - `sys::System`: system
 """
 function total_generation_rating(sys::System)
```

Comment:
we should add storage here

Reply:
Should the name of this function change?

Reply:
total_capacity_rating

Reply:
Ok, I support adding storage and changing to total_capacity_rating